Src/Application/CaseStudy.API/Controllers/MenuSettingsController.cs

```diff
@@ -0,0 +1,117 @@
+﻿using AutoMapper;
+using CaseStudy.Application.VM;
+using CaseStudy.Core.Contracts.IUnitOfWork;
+using CaseStudy.Core.DTO;
+using CaseStudy.Core.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CaseStudy.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MenuSettingsController : ControllerBase
+    {
+        private IMenuSettingsServices menuSettingsServices;
+        private IMapper mapper;
+        public MenuSettingsController(IMenuSettingsServices imenuSettingsServices,IMapper _mapper)
+        {
+
+            menuSettingsServices = imenuSettingsServices;
+            mapper = _mapper;
+        }
+
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<MenuSettings>>> GetMenuSettings()
+        {
+            try
+            {
+                var menuSettings =  await menuSettingsServices.GetMenuSettings();
+                if (menuSettings == null)
+                {
+                    return BadRequest("No Records Found");
```

Comment:
BadRequest is something you should return when UI did some mistake in sending the request https://http.dev/
we should not return when menuSettings is null